apps/frontend/src/hooks/pagination.ts

```diff
@@ -1,19 +1,36 @@
 import { useEffect } from 'react';
 import { useHistory, useLocation } from 'react-router-dom';
+import { viewParam, listViewValue } from '@asap-hub/routing';
 
-export const DEFAULT_PAGE_SIZE = 10;
+export const DETAILS_VIEW_PAGE_SIZE = 10;
+export const LIST_VIEW_PAGE_SIZE = 20;
 
-export const usePaginationParams = (pageSize = DEFAULT_PAGE_SIZE) => {
+export const usePaginationParams = () => {
   const history = useHistory();
   const searchParams = new URLSearchParams(useLocation().search);
   const currentPage = Number(searchParams.get('currentPage')) ?? 0;
 
+  const resetPaginationSearchParams = new URLSearchParams(searchParams);
+  resetPaginationSearchParams.delete('currentPage');
+
+  const listView = searchParams.get(viewParam) === listViewValue;
+  const listViewParams = new URLSearchParams(resetPaginationSearchParams);
+  listViewParams.set(viewParam, listViewValue);
+  const cardViewParams = new URLSearchParams(resetPaginationSearchParams);
```

Comment:
detail not card for wording consistency

Reply:
I think i snuck in another commit. They changed the designs on me

Reply:
Well, or if it's card view now, use that term everywhere :D

Reply:
So the default view is Card View. The new view is List View. I think the naming is consistent

Reply:
But there are still things named `detail` left